### PR TITLE
Update username in EIP-1319

### PR DESCRIPTION
**Pull request by njgheorghita**
Added the specification for ethPM uris. They've been an effective solution for handling packages so far - and are already implemented in the `ethpm-cli` and `Brownie`, and are also outlined in the [documentation](https://docs.ethpm.com/uris#registry-uris).

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1319 requires approval from one of (christophergewecke@gmail.com, nick.dandrea@consensys.net, piper@ethereum.org, nickg@ethereum.org)

**Comment by njgheorghita** [comment]
ping @gnidan @iamdefinitelyahuman @pipermerriam for input/confirmation. Specifically if all looks well, could @gnidan approve this pr - since my email listed in the pr is not the one associated with my github, so I can't get this merged.

**Comment by iamdefinitelyahuman** [comment]
I like it!  Looks very clean.  Thanks for pinging me on this.

File: EIPS/eip-1319.md [hunk 1]
```diff
@@ -166,6 +166,62 @@ release versions while allowing them to use any versioning schema they choose.
...
+
+#### `package_name`
+- Optional
+- String of the target package name
+
+#### `package_version`
+- Optional
+- String of the target package version
+- If the package version contains any [url unsafe characters](https://www.werockyourweb.com/url-escape-characters/), they **must** be safely escaped
+- Since semver is not strictly enforced by the ethpm spec, if the `package_version` is omitted from a uri, it is up to the tooling / framework to look up all available versions and ask the user which version they want to utilize.
```
Comment (pipermerriam) [inline EIPS/eip-1319.md]: This would probably be better to use the `**should**` language.  Something like:

> if the `package_version` is omitted from a uri, tooling **should** avoid guessing in the face of any ambiguity and present the user with a choice from the available versions.

File: EIPS/eip-1319.md [hunk 2]
```diff
@@ -166,6 +166,62 @@ release versions while allowing them to use any versioning schema they choose.
...
+
+`scheme://registry_address:chain_id/package_name@package_version/json_pointer`
+
+#### `scheme`
+- Required
+- Must be one of `ethpm` or `erc1319`
+
+#### `registry_address`
+- Required
+- This **should** be an ENS name whenever possible, or a `0x`-prefixed, checksummed address.
```
Comment (pipermerriam) [inline EIPS/eip-1319.md]: This reads as if an `ENS` name is preferable to the other formats.  Maybe reword as 

> This **should** be either an ENS name or a `0x`-prefixed, checksummed address.   ENS names are more suitable for cases where mutability of the underlying asset is acceptable and there is implicit trust in the owner of the name.  `0x` prefixed addresses are more preferable in higher security cases to avoid needing to trust the controller of the name.

**Comment by axic** [comment]
@njgheorghita is this still outstanding for merging?

**Comment by njgheorghita** [comment]
@axic Thanks for the reminder. It has all the necessary approvals afaik, it's just that the email address associated with my github account is not the one listed under my name as an author on this ERC. Would you be able to push the merge through?

**Comment by axic** [comment]
@njgheorghita do you want to change the author line to github username or another email so you can merge these draft changes?

**Comment by axic** [comment]
Can you also merge in master (or rebase)? We should check what the validator says now as this was creating quite a while ago.

File: EIPS/eip-1319.md [hunk 3]
```diff
@@ -166,6 +166,63 @@ release versions while allowing them to use any versioning schema they choose.
 
 Registries may offer more complex `read` APIs that manage requests for packages within a semver range or at `latest` etc. This EIP is agnostic about how tooling or registries might implement these. It recommends that registries implement [EIP 165](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md) and avail themselves of resources to publish more complex interfaces such as [EIP 926](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-926.md).
 
+## EthPM URIs
```
Comment (axic) [inline EIPS/eip-1319.md:169]: I'm not fully sure the best way is to include both the URI format and everything else in a single EIP. Would suggest for consideration to make this a separate EIP depending on this one.

**Comment by njgheorghita** [comment]
@axic I've gone ahead and moved the Ethpm URI spec to a new [ERC](https://github.com/ethereum/EIPs/pull/2942). I updated my username in this ERC just in case there are future edits that need to be made.

**Comment by MicahZoltu** [comment]
Should probably make this EIP require `2942`, since it references the path specification in here.

**Comment by MicahZoltu** [comment]
Though, it currently only references it casually in an example, without making any other mention anywhere in the specification of it.  Maybe we can just remove that one example and merge this (which ends up being just an author update)?

**Comment by njgheorghita** [comment]
@MicahZoltu Agreed, made the change, thanks!
